src/app/app.module.ts

```diff
@@ -231,7 +231,9 @@ export function initAuthentication(moduleConfig: OAuthModuleConfig, oauthService
       redirectUri: environment.authentication.redirectUri ? environment.authentication.redirectUri : window.location.origin + '/',
       silentRefreshRedirectUri: environment.authentication.silentRefreshRedirectUri,
       oidc: true,
-      requireHttps: environment.authentication.requireHttps
+      requireHttps: environment.authentication.requireHttps,
+      postLogoutRedirectUri: environment.authentication.postLogoutRedirectUri ?
```

Comment:
For consistency I'd recommend to set the post-logout URI also in the environment config and remove the fallback here.

Reply:
Thank you for the idea 👍 

I have set `postLogoutRedirectUri` to `window.location.origin + '/'` directly in the app module and removed the fallback.